pulsar-broker/src/main/java/org/apache/pulsar/broker/service/ServerCnx.java

```diff
@@ -1540,6 +1540,7 @@ private void buildProducerAndAddTopic(Topic topic, long producerId, String produ
                 getPrincipal(), isEncrypted, metadata, schemaVersion, epoch,
                 userProvidedProducerName, producerAccessMode, topicEpoch, supportsPartialProducer);
 
+        log.info("trying to add producer - {} to topic -{}", producerName, topicName);
```

Comment:
We already log about this, so I think we can remove this before merging.

https://github.com/apache/pulsar/blob/8cc979de411f9bca05ea95b16807f1860b79382e/pulsar-broker/src/main/java/org/apache/pulsar/broker/service/ServerCnx.java#L1374-L1377

Reply:
addressed